### DISCUSSION TITLE
Xdrip screen graph has unuseable interface

### DISCUSSION THREAD

**Discussion by hofno** [NightscoutFoundation/xDrip#3358] [Bug reports] [answered]
Hi! Im having terrible time trying to decode the readings on the graph (main) screen of the xdrip. Basically, everytime I start the app it shows me a cutout of the whole graph that is so detailed that it is completely unuseable in terms of informational value. I can zoom out of it but when I restart the app or just leave it and come back to it, it defaults back to ultrazoom. I am attaching screenshots for the reference. Tried fixing it in the settings but to vein. also used xdrip+ on a different phone till recently and had no such issues at all.

Techincal data: 
im using latest version of xdripJH  @jakobheyman (was using latest xdrip+ couple of days back with same results) connected to fresstyle libre 2 EU over bluetooth with oop2 algorythm app. my phone is redmi 12C MIUI Global 14.0.7 on Android 13

![xdripstatus](https://github.com/NightscoutFoundation/xDrip/assets/24904174/4d2f6bd1-9b3c-4178-bf5b-59987594945f)
![xdrip screen2](https://github.com/NightscoutFoundation/xDrip/assets/24904174/4e4f3ace-3361-478a-8f65-274a1978eb39)
![xdrip screen1](https://github.com/NightscoutFoundation/xDrip/assets/24904174/1175acb1-492e-4f22-8d27-06568100b4cf)

**Comment by hofno** [comment]
just to clarify what i mean by that "zoom" im adding another screenshot. thanks for your suggestions in advance
![xdrip screen2](https://github.com/NightscoutFoundation/xDrip/assets/24904174/9548c366-1cd1-45fe-b607-544cbbe320bc)

**Reply by PhilTU** [reply, marked as answer]
What is your y-axis maximum setting at ? You can customize it to 22.2 in settings which might have an effect.

**Reply by hofno** [reply]
thank you. that did it :)

**Comment by Navid200** [comment]
This is how you can control the horizontal or x axis: https://navid200.github.io/xDrip/docs/TimeButtons.html

You have high values.  xDrip has no option to extend the vertical or y axis so that it can show those.
This is more about that: https://navid200.github.io/xDrip/docs/Display/CompressedScreen.html

Please let me know if you are asking about something else.

**Reply by hofno** [reply]
Hey! Thank you for your comment and the links. Unfortunately, your suggestion dont solve the issue.

My problem is not the horizontal/time/x axis. Rather, its the vertical axis. In my opinion, its not the high readings that cause the issue. Rather, it is the default "zoom in" of what exactly is displayed that renders readings human-unreadable. I can fix this by zooming out on the lover graph plane manually, but i need to do this every single time after restarting xdrip or even switching between apps, which is insanely inconvenient. The solution would be to force the lower graph to settle on the whole length of y axis, rather than it's tiny fraction. Please reffer to the screenshot with description in my  comment above. In the link where thst you provide the zoom of the lower graph is set "correctly" but that is not my case and i dont know how to change that. Would you have a suggestion here?
Thank you 🙏
![Screenshot_20240223_100316](https://github.com/NightscoutFoundation/xDrip/assets/24904174/85679569-0e73-4528-839d-e47344142a57)

**Comment by Navid200** [comment]
Can you post a screenshot of the classic status page?
This one: https://navid200.github.io/xDrip/docs/xDrip-Version.html

**Reply by Navid200** [reply]
The xDrip vertical (y) axis will never zoom in.  If there are no high values for 24 hours, you can zoom in by entering a preferred range.  But, as soon as you have reading outside that range, xDrip automatically extends the vertical axis so that xDrip can show the readings.
But, xDrip does not pan and does not zoom in to exclude any readings.

I would like to know what version of xDrip you are using.

**Reply by hofno** [reply]
@Navid200 hello
i posted the screen in the first original question.

**Comment by jakobheyman** [comment]
I believe the problem is associated with some changes of the app used (xdripJH) which is a modified fork of xDrip+, and I guess the problem would not appear (at least not in the same way) in the original xDrip+ app.

**Reply by Navid200** [reply]
Yes, that's what I suspected and that's why I asked to see the version.
I wish people helped us improve xDrip instead of creating their own versions.

**Reply by hofno** [reply]
@Navid200 the version together with explenation and screenshot of my screen is provided in my first post. what i wish is that people read through questions twice before starting axusations. the issue was same with regular xdrip as far as i remember. changing settings of y axis to different values did it. nothing to do with removing flags at all. i provided your screenshot modified so you know what im talking about and have a comparison to screenshot i posted in my original question. not shure that helped you much, but thanks anyway, problem is sloved now

**Comment by Navid200** [comment]
Ask for support from the person whose software you are using.

**Reply by Navid200** [reply]
Alternatively, you can use the official xDrip:
https://navid200.github.io/xDrip/docs/Installation_page.html

**Reply by hofno** [reply]
yes did that too. hes here in the discussion as well

**Reply by Navid200** [reply]
The official xDrip will never zoom in vertically.  

You are using an unofficial release and you are asking questions in the official release forum.
I didn't know you were using an unofficial release.  
Please be considerate and ask for help from the person who has created the version you are using.

**Reply by hofno** [reply]
yes i did ask at both forums since i thought this would be a common feature. its solved now so no need to take this any further. im stating the version im using in my first post though so im not playing hide and seek here either. just reading through the question tiwce would solve a lot of misunderstanding already...

**Reply by Navid200** [reply]
I'm sorry I misunderstood and made a mistake.  I assumed you were using the official release and when I provided my suggestions they were all based on the assumption that you were using the official release.

I am glad the problem is solved.  Great.
